Makefile

```diff
@@ -125,7 +127,8 @@ release:
 	cd artifacts && for x in shield-server-*; do \
 	  cp -a ../web/htdocs $$x/webui; \
 	  mkdir -p $$x/webui/cli/linux; cp ../artifacts/shield-linux-amd64   $$x/webui/cli/linux/shield; \
-	  mkdir -p $$x/webui/cli/mac;   cp ../artifacts/shield-darwin-amd64  $$x/webui/cli/mac/shield; \
+	  mkdir -p $$x/webui/cli/mac-amd64;   cp ../artifacts/shield-darwin-amd64  $$x/webui/cli/mac-amd64/shield; \
+	  mkdir -p $$x/webui/cli/mac-arm64; cp ../artifacts/shield-darwin-arm64 $$x/webui/cli/mac-arm64/shield; \
```

Comment:
Perhaps it would be cleaner to do mac/{amd64,arm64} as subdirectories?

Reply:
Additionally I think we should stick to uname output and have it be `darwin/$ARCH`

Reply:
More clear statement: it should be `darwin/{amd64/arm64}` not `mac-*`

Reply:
I'm fine with that too.